### ISSUE TITLE
Allow switching to different frontends

### ISSUE THREAD

**Issue by muppeth** [BDX-town/Mangane#123]
I noticed that when using Mangane you can't use other frontends such as pleroma-fe, or mastodon-fe. When on pleroma-fe you can use `/web` endpoint to swtich. I dont know if this is mangane related issue or it needs to be changed on akkoma (so please let me know), It would be fantastic to be able to switch frontends within Mangane's side bar.

**Comment by Cl0v1s** [comment]
Good point. Do you have a screenshot of that /web endpoint you are speaking of ? :)

**Comment by muppeth** [comment]
it's mastodon-fe. https://akkoma.dev/AkkomaGang/masto-fe  I think it's working to some extend with manfane as primary frontend though i need to check a bit more. 
I should create a feature request on akkoma to perhaps be more flexible when it comes to frontends and instead of tweaking webserver config, have a toggle in user preferences saved in database. In that way users could choose their desired frontend without the need to remember the url of the frontend of their choice.

**Comment by Cl0v1s** [comment]
Indeed, I'm not sure if there is any specs regarding url and how to make various frontends work together.
Most of the instance I see offering multiple frontends do it by using multiple subdomain for example `mangane.domain.tld`. 

I'll into that

**Comment by Cl0v1s** [comment]
Did you achieve to resolve your problem with using subdomains ?

**Comment by arm77** [comment]
How can I access the default admin-fe frontend when Mangane is the primary frontend?

**Comment by arm77** [comment]
> How can I access the default admin-fe frontend when Mangane is the primary frontend?

Answering my own question - use the `pleroma/admin` endpoint. However, it fails in my case since the admin user has MFA/2FA enabled which is issue #115

**Comment by lapineige** [comment]
> How can I access the default admin-fe frontend when Mangane is the primary frontend?

I believe that's an issue too : if you don't know by heart (or guess) the frontends endpoints, you can't reach them from either any of the frontends.

There should be a link somewhere to reach admin frontend, shouldn't it ?

**Comment by Cl0v1s** [comment]
There is an ongoing issue about that :)
(Don't have the link rn tho, but it's in the next release I believe)

**Comment by Cl0v1s** [comment]
I think it's more an issue with conflicting routes in various frontends

**Comment by pablo-abc** [comment]
I was just experimenting with Akkoma and Mangane. Got to this issue as well.

One detail that I found is that if I go to '/web' directly on a fresh private tab, the mastodon fe works as expected. But if Mangane has loaded at any moment I suddenly can't reach '/web' anymore.

Leads me to believe it has to do with the service worker caching the routes?

**Comment by thomasjwebb** [comment]
I'm having this issue too and I think it's a major security issue that an admin has to disable mfa to be able to access the admin-fe.
